controller/controller_test.go

```diff
@@ -20,11 +21,6 @@ import (
 func TestNewControllers(t *testing.T) {
 	t.Parallel()
 
-	newCtrlFuncs := map[string]func(*config.Config) (Controller, error){
-		"readwrite": NewReadWrite,
-		"readonly":  NewReadOnly,
-	}
-
```

Comment:
I couldn't figure out a way to keep this pattern! :(

Reply:
haha no worries, it was just a way i grouped those tests at the time, things will definitely evolve and it might make more sense to separate testing the two in the future.